### PR TITLE
add interactive examples to gallery

### PR DESCRIPTION
**Pull request by rc**
One by one all examples work, but together the script stops in the middle - maybe some interaction with pyvista event loop(?)

### CODE REVIEW

**Comment by rc** [comment]
Now the weird stopping looks related to the parallel examples that use petsc.

**Comment by rc** [comment]
The current gallery can be seen at https://sfepy.org/doc-devel/examples/gallery.html.

**Comment by rc** [comment]
I will skip the DG examples for now, because they need gmsh and I do not know how to use it non-interactively. So it is ready. Ping @vlukes.

**Comment by vlukes** [comment]
... and:

```  File "/home/vlukes/tmp/rc/sfepy/./tools/gen_gallery.py", line 1221, in <module>
    main()
  File "/home/vlukes/tmp/rc/sfepy/./tools/gen_gallery.py", line 1208, in main
    generate_images(images_dir, examples_dir, pattern=options.pattern)
  File "/home/vlukes/tmp/rc/sfepy/./tools/gen_gallery.py", line 846, in generate_images
    _apply_commands(custom_options, images_dir)
  File "/home/vlukes/tmp/rc/sfepy/./tools/gen_gallery.py", line 765, in _apply_commands
    shutil.copy(val, images_dir)
  File "/usr/lib/python3.10/shutil.py", line 417, in copy
    copyfile(src, dst, follow_symlinks=follow_symlinks)
  File "/usr/lib/python3.10/shutil.py", line 254, in copyfile
    with open(src, 'rb') as fsrc:
FileNotFoundError: [Errno 2] No such file or directory: 'output/frequencies.png'
```

**Comment by rc** [comment]
Thanks, fixed. I had stale files in my output directories.

**Comment by vlukes** [comment]
It works now!

It would be nice to show also `linear_elasticity/truss_bridge.py` and/or `linear_elasticity/truss_bridge3d.py` examples in the gallery. What do you think?

**Comment by rc** [comment]
> It would be nice to show also `linear_elasticity/truss_bridge.py` and/or `linear_elasticity/truss_bridge3d.py` examples in the gallery. What do you think?

They are there, but they are broken in this branch - the fix is in the master.

**Review by vlukes — APPROVED** [review]

File: sfepy/scripts/resview.py [hunk 1]
```diff
@@ -625,6 +626,8 @@ def print_camera_position(plotter):
     cp = nm.array([k for k in plotter.camera_position]).ravel()
     cp = ','.join(['%g' % k for k in cp])
     print(f'--camera-position="{cp}"')
+    with open('cp.txt', 'w') as fd:
```
[the remark this answers is not in the document]
Comment (rc) [reply sfepy/scripts/resview.py:629]: No, it is a debug code that sneaked in. Sometimes the 'c' button does not print anything, so I use this as a workaround...

**Pull request merged**